no-vary-search-security-privacy-questionnaire.md

```diff
@@ -0,0 +1,77 @@
+# No-Vary-Search Security & Privacy Questionnaire
+
+> 01.  What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+None. This feature allows websites to expose information about themselves to the browser, but not the other way around.
```

Comment:
I think it's innocuous, but anything that affects caching does at least provide the opportunity for exposure to some information about the history of previous fetches to that cache (because a request might not be sent in the event of a cache hit, the origin server can make an inference).

I think the differential privacy surface is very small  here, though, and mitigations that apply to caches remain in force with `No-Vary-Search` support as far as I can immediately tell.

Reply:
Fair; added.